chicken_dinner/constants.py

```diff
@@ -67,6 +73,9 @@
     "Range_Main": [204800, 204800],
     "Baltic_Main": [819200, 819200],
     "Summerland_Main": [204800, 204800],
+    "Tiger_Main": [819200, 819200],
+    "Chimera_Main" : [819200, 819200],
+    "Heaven_Main" : [307200, 307200],
```

Comment:
I've not played this game in a long time. Is this correct? Some googling shows this is a 1x1 map.

Reply:
I've fixed the typos; Chimera_Main (Paramo) should be 3x3, and Heaven (Haven) is 1x1. Paramo is a bit of a strange one though; the map locations (towns and other named locations) rotate from match to match. Names etc. stay the same, but move to different parts of the map. Kinda like School from Erangel suddenly being on the island. Not sure how well that will work with replays, but the coordinates stay the same I should think; just that using the same basemap will make for some wierd renders. 

Going to have  a look at chicken_dinner.assets as well, and then I will commit the changes.